src/generators/number/index.ts

```diff
@@ -1,16 +1,25 @@
-import {Chance} from 'chance';
+import { Chance } from 'chance';
 
 /**
- * Generates a random `number`, optionally between `lowerInclusive` and `upperExclusive`.
+ * Generates a random `number`, optionally between `lowerInclusive` and `upperExclusive` with `fixed` number of digits after the decimal.
  */
 const createNumberGenerator = (chance: Chance.Chance) => (
   lowerInclusive = -1000000,
   upperExclusive = 1000000,
-): number =>
-  chance.floating({
+  fixed = 4,
+): number => {
+  const EPSILON = 0.0000001;
+  const MAX_PRECISION = 6;
```

Comment:
move this out of the function? no need to re-declare each time

Reply:
Addressed in 910d483